api/core/model_providers/models/embedding/azure_openai_embedding.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, model_provider: BaseModelProvider, name: str):
             max_retries=1,
             **self.credentials
         )
+        # TODO load price_config from configs(db)
```

Comment:
Because there may be discounts available through partnerships, the actual price and the listed price may not necessarily be the same. so, I think it is better for config by user self.